docs/qcvv/xeb_theory.ipynb

```diff
@@ -1,14 +1,5 @@
 {
```

Comment:
<p>this renders strange in reviewnb.com  (&lt;span style...&gt;).  Could you confirm this is fine elsewhere?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I think it renders strange only in the diff between old and new in the revewnb notebook. See the attached screenshot for a standalone view. </p><p> </p><p>I can confirm that it renders as expected in jupyter / colab. </p>